dictionary/task_02.py

```diff
@@ -0,0 +1,10 @@
+grades = {
+    "Alice": 85,
+    "Bob": 92,
+    "Charlie": 88,
+    "Diana": 95,
+    "Eve": 78
+}
+
+for student in grades:
+    print(f"{student}: {grades[student]}")
```

Comment:
This way we avoid additionally extracting grades

```suggestion
for student, grade in grades.items():
    print(f"{student}: {grade}")
```

Reply:
Got it, thank you)